In [2]:
1 - 0.0038

0.9962

# Question 2 
## Part 1
$$ p(early) = 0.02$$
$$ p(early)^2 = (0.02)^2 \rightarrow 0.04 $$ 
## Part 2 
$$ p(early \space\cap\space late) = 0.19*0.02 \rightarrow 0.0038$$
$$ p(on \space time) 1 - 0.0038 = 0.9962$$
## Part 3
For part 1, we already know the probability that the bus is going to be early for a day. Since the fact that the bus being early the next day does not impact the bus being early today, we can calculate the probability of the bus being early for the next two days by just squaring the probability of the bus being early. 

For part 2, we know that the probability of the bus being early and the bus being late are independent from each other. Therefore, multiplying the two probabilities gives us the probability of the bus being early and delayed. One minus this probability, give us the probability that the bus is neither early and neither late. Another way to interpret this, is to say that it is the probability that the bus is on time. 